x-pack/plugins/observability/server/domain/services/compute_error_budget.test.ts

```diff
@@ -13,8 +13,8 @@ import { computeErrorBudget } from './compute_error_budget';
 import { toDateRange } from './date_range';
 
 describe('computeErrorBudget', () => {
-  describe('for occurrences based SLO', () => {
-    describe('with rolling time window', () => {
+  describe('for rolling time window', () => {
```

Comment:
💬  encapsulated with rolling time window first since both occurrences and timeslices behave the same way